### DISCUSSION TITLE
Premades for BG and a better balance to the teams

### DISCUSSION THREAD

**Discussion by Wayran-is-typing** [chromiecraft/chromiecraft#3258] [Suggestions]
On this days one of the most disgusting things from battlegrounds is watch a lot of AFK or useless people doing nothing for get the victory based on any strategy from the leader of the group or anyone who just want to win the game (for farming honor points, achievementes or just getting some fun killing ppl). Maybe give a chance or a test weekend for the guilds of CC to able their members to join togethers a battleground could make a lot of ppl more interested on pvp for the unique reason of share time together, like it happens on the raiding days.

With balanced teams i mean for example disable the possibility of four/five hunters on the same team and none of them on the other one, basically if all the hunters are BM is like a 99% chances of a free win for that team. The same for the healers, in a WSG if one team has two healers and the other one none there is nothing to do.

**Comment by 55Honey** [comment]
Hello Josarth,
thanks for your suggestion.

We've had an open issue for a long time about enabling the group-queue for BGs. It just didn't work. We have recently applied an update which allowed for group queue, but the code isn't stable enough yet, so it was deactivated for further development. We will consider re-adding the group queue, when it works better.

**Reply by Wayran-is-typing** [reply]
ahh ok, ty!
